src/OI.cpp

```diff
@@ -65,8 +65,10 @@ OI::OI()
 {
   // Process operator interface input here.
 
+/*    We'll need these later but keep this like this for now.
   static const double kRPMNudge = 10.0;
-  static const double kPositionNudge = 0.;
+  static const double kPositionNudge = 0;
```

Comment:
Double and float should always include a decimal point. This is technically assigning an integer to a double. The integer is promoted to a double. If the number includes the decimal point, then a double is assigned to a double, not conversion is required.